olive/evaluator/olive_evaluator.py

```diff
@@ -337,9 +338,11 @@ def _inference(
             # keep as numpy or torch arrays
             preds.append(outputs.cpu())
             targets.append(labels.cpu())
+            logits.append(result.cpu())
         preds = torch.cat(preds, dim=0)
         targets = torch.cat(targets, dim=0)
-        return preds, targets
+        logits = torch.cat(logits, dim=0)
+        return preds, targets, logits
```

Comment:
do we need define a class to contains all of them? is it possible that we need add other values in future?

Reply:
something like https://huggingface.co/docs/transformers/main/main_classes/output#transformers.utils.ModelOutput

Reply:
is this line impacted? https://github.com/microsoft/Olive/pull/460/files#diff-816af95cf16e31610395bb20947f1d73cc1ca67355964e074a8f55375c8139a7R123

Reply:
ditto https://github.com/microsoft/Olive/blob/main/test/unit_test/assets/user_script.py#L8C5-L8C16

Reply:
Updated with tuple(preds, logits)